### PR TITLE
Jesus Take the Wheel

### PR DESCRIPTION
**Pull request by steets250**

### CODE REVIEW

File: src/auth/containers/requireStoreAccess.tsx [hunk 1]
```diff
@@ -13,10 +13,12 @@ const withStoreAccess = (Component: React.FC) => (props: { [key: string]: any })
...
-    if (state === UserState.PENDING || !(emailDomain === 'ucsd.edu' || emailDomain === 'acmucsd.org')) {
-      redirectHome();
-    } else {
-      setPermitted(true);
+    if (email) {
+      if (state === UserState.PENDING || !(emailDomain === 'ucsd.edu' || emailDomain === 'acmucsd.org')) {
+        redirectHome();
+      } else {
+        setPermitted(true);
+      }
```
Comment (StormFireFox1) [inline src/auth/containers/requireStoreAccess.tsx:21]: How come there's no email? When does that happen?

**Review by StormFireFox1 — COMMENTED** [review]
Nit, and I'll approve!

File: src/auth/containers/requireStoreAccess.tsx (hunk 1 above)
Comment (steets250) [reply src/auth/containers/requireStoreAccess.tsx:21]: `requireStoreAccess` is nested within `requireAccess`, which ensures that the user is signed in and their data has loaded. By checking that the email is defined, it ensures that we don't throw an invalid email error before the user profile has loaded.

Comment (StormFireFox1) [reply src/auth/containers/requireStoreAccess.tsx:21]: Yep, makes sense!

**Review by StormFireFox1 — APPROVED** [review]
LGTM!

**Pull request merged**